esp32_s3_korvo_2/CMakeLists.txt

```diff
@@ -1,7 +1,16 @@
+#IDF version is less than IDF5.0
+if("${IDF_VERSION_MAJOR}.${IDF_VERSION_MINOR}" VERSION_LESS "5.0")
+    set(SRC_VER "esp32_s3_korvo_2_idf4.c")
+    set(PRV_REQ fatfs esp_lcd spiffs)
+else()
+    set(SRC_VER "esp32_s3_korvo_2_idf5.c")
+    set(PRV_REQ fatfs esp_lcd esp_adc spiffs)
```

Comment:
Since *include/bsp/display.h*  uses `#include "esp_lcd_types.h"` and *include/bsp/esp32_s3_korvo_2.h* uses `adc_oneshot_unit_handle_t`, maybe we should take `esp_lcd esp_adc` into `REQUIRES`?